dotnet-docs/windows-installation.md

```diff
@@ -10,48 +10,55 @@ position: 2
 
 # Report Server for .NET: Installation on Windows
 
-The Report Server for .NET (`RS.NET`) is currently distributed along with the installer for the .NET Framework 4.6.2. By default, the installer does not install RS.NET. Users must click `Customize` to install RS.NET.
+Starting with 2025 Q4, a dedicated Report Server for .NET (RS.NET) installer is available. This article explains how to install RS.NET on Windows and covers the available installation scenarios.
 
-## Installation Process
+> Prior to 2025 Q4, RS.NET was distributed with the [.NET Framework 4.6.2 installer]({%slug installation%}). If you use an older version that features this combined installer, make sure to click **Customize** during installation to include RS.NET.
 
-The RS.NET is an ASP.NET Core web application and its installation on the IIS requires the `ASP.NET Core Hosting Bundle` as explained in the Microsoft article [Host ASP.NET Core on Windows with IIS](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-8.0). The installation wizard will show a warning if the module is not installed. The user can continue the installation even when the module is not found.
+## Prerequisites
 
->note Known issue: the detection returns false negative results on machines having `Windows 11`, `Windows Server 2016` and `Windows Server 2022`. This is fixed and will be included in our next release.
+Before installing RS.NET, ensure that:
 
-The installer will configure the ports for installing the RS.NET and RS.NET Service Agent, taking the available ports from 80 upwards:
+- Your system [meets the requirements]({%slug system-requirements%})
+- [ASP.NET Core Hosting Bundle](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-8.0) for IIS deployment is installed.
 
-![Use the button Customize to allow installing the Report Server for .NET](../images/rs-net-images/rs-customize.png)
+> The installation wizard displays a warning if the ASP.NET Core Hosting Bundle is not available. Ignoring it makes the installed Report Server non-functional.
```

Comment:
I would use a modal verb here: "...might make the installed RS non-functional", because we've had scenarios on few OS-es where the ASP.NET Core hosting bundle was installed, but the installer failed to detect it.
Also, can we extend the sentence, adding that the ASP.NET Core bundle can be installed after installing the RS.NET or it might confuse the clients?